tests/workload/network.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package workload
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+)
+
+type TestNetwork interface {
+	Nodes() []TestNode
+	Configuration() TestNetworkConfiguration
+}
+
+type TestNode interface {
+	SubmitTxs(context.Context, []*chain.Transaction) error
+	ConfirmTxs(context.Context, []*chain.Transaction) error
+	GenerateTx(context.Context, []chain.Action, chain.AuthFactory) (*chain.Transaction, error)
```

Comment:
Can we remove `TestNode` and support a single `ConfirmTxs` call on `TestNetwork` instead of the node?

We should leave it to each environment (integration/e2e) to handle this since submitting a tx to a single node in integration will result in a different outcome than submitting a tx to a single node in the e2e environment.

Each environment should enforce that marking a tx as confirmed indicates all nodes have marked it as confirmed, so that we can check the state by making API calls directly to the URIs reported by the `TestNetwork`.

Reply:
done